src/phrases/phrases-nb.ts

```diff
@@ -1504,6 +1504,7 @@ const phrases = {
     getExisting: 'Eller hent eksisterende ressurs',
     inProgress: 'Denne artikkelen er påbegynt',
     addPriority: 'Sett artikkel som prioritert',
+    publishRequired: 'Dette krever delpublisering av taksonomi',
```

Comment:
Trur vi kan konkretisere enda litt meir. 
```suggestion
    publishRequired: 'Endring av tittel krever delpublisering av taksonomi',
```